lib/thredded/private_topics_search.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require_dependency 'thredded/search_parser'
+module Thredded
+  class PrivateTopicsSearch
```

Comment:
This looks almost identical to TopicsSearch, perhaps they can both inherit from TopicsCommonSearch and share most of the implementation?